packages/tempus/src/Tempus_IntegratorBasic_impl.hpp

```diff
@@ -97,16 +95,12 @@ void IntegratorBasic<Scalar>::setStepper(
   using Teuchos::RCP;
   using Teuchos::ParameterList;
 
-  if (stepper_ == Teuchos::null) {
```

Comment:
Is there a reason for eliminating the guard if the stepper is non-null?

Reply:
Maybe we should leave it as is.  I thought there was some wrong logic here, but it is probably ok.  I am planning dropping this function anyway, as a function to set the Stepper from models as the argument is confusing.  If the function is setStepper(), it should set the stepper, not set the models on the stepper. 

I will return to the original coding.

Also I am not liking how we set the steppers for StepperOperatorSplit steppers, i.e., give a list of steppers and a list of models and make them match.  I think we should remove this functionality and exclusively go with completely setup a stepper with its model, and then add it to the StepperOperatorSpit stepper via addStepper().  That way it is clear what models go with what stepper.  thoughts?

Reply:
Ah, yes.  Stepper::createSubSteppers() with a single argument was a no-op function!  So the function did nothing when stepper_ != null.  The whole thing is ugly and not tested.  Thus I think we just need to remove it in a future PR.

Reply:
Let's not worry about this in this PR.  I would suggest approving this PR and make fixes in another PR.